dll/win32/shell32/res/rgs/controlpanel.rgs

```diff
@@ -73,3 +74,28 @@ HKCR
 		}
 	}
 }
+
+HKCR
+{
+	NoRemove CLSID
+	{
+		ForceRemove {0DF44EAA-FF21-4412-828E-260A8728E7F1} = s 'Taskbar and Start Menu'
+		{
+			val '{305CA226-D286-468e-B848-2B2E8E697B74} 2' = d '1'
+			val InfoTip = e '@%%SystemRoot%%\system32\SHELL32.dll,-30348'
```

Comment:
We cannot use `%MODULE%` here instead of the shell32 hardcoded path?
(Not a problem for the PR per se: I can see the same thing already exists for the `'Control Panel'` entry much above.)

Reply:
Perhaps but the result is not the same. Meaning, %MODULE% expands to a hardcoded path. This is using %SystemRoot% and an expandable registry string.

I'm just doing what Windows does here, if that is best can be debated. At least here for a control panel item, I don't think expanding the string is too costly. On the other had, are you ever going to change the name of your Windows directory?

Reply:
> On the other had, are you ever going to change the name of your Windows directory?

Exactly, nobody would do that otherwise they would just break their install.